### PR TITLE
Resolve conflict for #8235

### PR DESCRIPTION
**Pull request by vparfonov**
see details in #8235

### CODE REVIEW

**Comment by vparfonov** [comment]
ci-test

File: agents/ls-typescript/src/main/resources/installers/1.0.1/org.eclipse.che.ls.typescript.script.sh [hunk 1]
```diff
@@ -143,7 +143,7 @@ elif echo ${LINUX_TYPE} | grep -qi "opensuse"; then
     test "${PACKAGES}" = "" || {
         ${SUDO} zypper install -y ${PACKAGES};
     }
-
+FF
```
Comment (tolusha) [inline agents/ls-typescript/src/main/resources/installers/1.0.1/org.eclipse.che.ls.typescript.script.sh]: Typo?

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ShowHiddenFilesAction.java [hunk 2]
```diff
@@ -65,8 +72,15 @@ public void updateInPerspective(@NotNull ActionEvent event) {
 
   @Override
   public void actionPerformed(ActionEvent e) {
-    boolean isShow = projectExplorerPresenter.isShowHiddenFiles();
-    projectExplorerPresenter.showHiddenFiles(!isShow);
+    //    boolean isShow = projectExplorerPresenter.isShowHiddenFiles();
```
Comment (tolusha) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ShowHiddenFilesAction.java]: Any reasons to keep comments?

**Review by tolusha — APPROVED** [review]

File: agents/ls-typescript/src/main/resources/installers/1.0.1/org.eclipse.che.ls.typescript.script.sh (hunk 1 above)
Comment (vparfonov) [reply agents/ls-typescript/src/main/resources/installers/1.0.1/org.eclipse.che.ls.typescript.script.sh]: yes, thx

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ShowHiddenFilesAction.java (hunk 2 above)
Comment (vparfonov) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/ShowHiddenFilesAction.java]: any reason will fix it

**Comment by vparfonov** [comment]
ci-test-docker-single

**Review by evidolob — APPROVED** [review]

**Comment by vparfonov** [comment]
ci-test

**Comment by musienko-maxim** [comment]
After selenium session 3 tests fell. But after rerunning they have passed. So existed selenium tests did not find new regression

**Review by musienko-maxim — APPROVED** [review]

**Pull request closed without merging**